### ISSUE TITLE
[BUG] Automatic installation from sources failed

### ISSUE THREAD

**Issue by lfjdhr123** [polycube-network/polycube#328]
## Automatic installation from sources failed
During my installation process, I followed the instruction [Automatic installation from sources: Released version (stable)](https://polycube-network.readthedocs.io/en/latest/installation.html). The installation failed.

## To Reproduce
1. wget https://github.com/polycube-network/polycube/archive/v0.9.0-rc.zip
2. unzip v0.9.0-rc.zip
3. cd polycube-0.9.0-rc
4. ./scripts/install.sh
5. see the error:
```
Install polycube
+ cd /home/fangjia/workspace/polycube-0.9.0-rc/scripts/..
+ '[' '' == true ']'
+ '[' '' == update ']'
+ mkdir -p build
+ cd build
+ git log -1
fatal: not a git repository (or any of the parent directories): .git
++ error_message
++ set +x
Error during installation
```

## Expected behavior
Polycube should be installed

## Please tell us about your environment: 
1. OS details: `cat /etc/*-release` 
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.5 LTS"
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```
2. Kernel details: `uname -a`
```
Linux fangjia 5.4.0-42-generic #46~18.04.1-Ubuntu SMP Fri Jul 10 07:21:24 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```
3. Polycube Version: `polycubectl version`
```
v0.9.0-rc
```
4. Polycube Log: Attach the log from `/var/log/polycube/polycubed.log`
```
N?A
```

## Additional context
From my understanding, it is failed since the extracted project directory is not git repository. So I changed another way:
```
	1. git clone https://github.com/polycube-network/polycube
	2. cd polycube
	3. git checkout tags/v0.9.0-rc
	4. git submodule update --init --recursive
	5. ./scripts/install.sh
```
Then the installation succeeded. Not sure my way is correct or not,

**Comment by acloudiator** [comment]
@lfjdhr123 It's a duplicate from #322 and latest release would have fix this issue.

Please follow the below comment.
https://github.com/polycube-network/polycube/issues/322#issuecomment-691146415

**Comment by frisso** [comment]
Fixed by #333
